src/main/kotlin/app/revanced/patches/youtube/layout/pivotbar/shortsbutton/patch/ShortsButtonRemoverPatch.kt

```diff
@@ -63,7 +64,7 @@ class ShortsButtonRemoverPatch : BytecodePatch(
         val enumScanResult = fingerprintResults[0]
         val buttonViewResult = fingerprintResults[1]
 
-        val enumHookInsertIndex = enumScanResult.startIndex
+        val enumHookInsertIndex = enumScanResult.startIndex + 2
```

Comment:
Will this not cause issues with existing versions of YouTube?

Reply:
Nope...this is based on a fingerprint equal on 17.36.37 and 17.41.37.

Reply:
Please add the changes to the fingerprint.